### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by dunkgray**
Create an ard_pbs script and optionally call it. Plus there are some hacky shell example scripts.

### CODE REVIEW

File: region_code_filter.py [hunk 1]
```diff
@@ -286,8 +300,13 @@ def _do_parent_search(dc, product, days_delta=0):
         processed_ard_scene_ids = {chopped_scene_id(s) for s in processed_ard_scene_ids}
     else:
         processed_ard_scene_ids = None
+        _LOG.info(
+           "Child ARD not know for product: (%s)", product
```
Comment (sixy6e) [inline region_code_filter.py:304]: Probably just a minor grammatical error.  But the sentence is not that clear to me.

File: region_code_filter.py [hunk 2]
```diff
@@ -370,6 +406,37 @@ def get_landsat_level1_file_paths(
...
+        ard_params.append("--" + key)
+        # Make path strings absolute
+        if key in ('logdir', 'pkgdir'):
+            value = Path(value).resolve()
+        ard_params.append(str(value))
+    ard_arg_string = " ".join(ard_params)
+    return ard_arg_string
+
+
+def make_ard_pbd(**ard_click_params):
```
Comment (sixy6e) [inline region_code_filter.py:427]: *make_ard_pbd* or *make_ard_pbs* ?

File: region_code_filter.py [hunk 3]
```diff
@@ -461,26 +528,89 @@ def get_landsat_level1_file_paths(
...
+    :return: list of scenes to ARD process
+    """
+    workdir = Path(workdir).resolve()
+    # set up the scene select job dir in the work dir
+    jobid = uuid.uuid4().hex[0:6]
+    jobdir = Path(os.path.join(workdir, FMT2.format(jobid=jobid)))
+
+    #
+    print("Job directory: " + str(jobdir))
+    log_filepath = os.path.join(jobdir, LOG_FILE)
```
Comment (sixy6e) [inline region_code_filter.py:605]: any reason to be using `os.path.join` as opposed to `workdir.joinpath`?

File: region_code_filter.py [hunk 4]
```diff
@@ -461,26 +528,89 @@ def get_landsat_level1_file_paths(
...
+    if not os.path.exists(jobdir):
```
Comment (sixy6e) [inline region_code_filter.py:606]: similar to comment above.  If it is a `Path` object, its then probably cleaner to use the `.exists` method.

File: region_code_filter.py (hunk 2 above)
Comment (dunkgray) [reply region_code_filter.py:427]: Ha ha.  This is what I like about code reviews.  At least the function is returning a pbs. [I'll fix this!]

**Comment by dunkgray** [comment]
Thanks for the suggestions Josh.  I have added them.

**Pull request merged**